chain-impl-mockchain/src/accounting/account/account_state.rs

```diff
@@ -87,6 +94,39 @@ pub struct AccountState<Extra> {
     pub extra: Extra,
 }
 
+#[cfg(any(test, feature = "property-test-api"))]
+mod test_impls {
```

Comment:
I would prefer to store all this code inside the `tests` module as all other implementations are done.

Reply:
For these changes, what do you think about:
```rust
mod tests {
  // quickcheck impls

  mod pt {
    // proptest impls
  }
}
```
where `mod pt` exists to create a namespace to avoid collisions with `quickcheck`?

Reply:
Oh, I we will have a collisions, so you can leave it as it is, but maybe rename from `test_impls` to `prop_impls`